tests/Unit/DependencyInjection/Fixtures/invalid/redis.yml

```diff
@@ -3,3 +3,5 @@ maba_gentle_force:
         host: localhost
         service_id: abc
         prefix: my_prefix
+        parameters: []
```

Comment:
This must fail without any modifications to the original file - if such configuration failed before, it must fail afterwards.